pkg/compactor/compactor.go

```diff
@@ -62,7 +62,7 @@ var (
 			return nil, nil, err
 		}
 
-		planner := compact.NewTSDBBasedPlanner(logger, cfg.BlockRanges.ToMilliseconds())
+		planner, _ := NewPlanner(logger, cfg.BlockRanges.ToMilliseconds(), "default")
```

Comment:
consider adding the config option, with a default value of "thanos", and see what the community says

Reply:
remember to check for error, as you suggested